js/repl/public/index.html

```diff
@@ -0,0 +1,52 @@
+<!doctype html>
```

Comment:
Interesting. So the only tricky part is making it work nicely with dev workflow. (Adding front matter would break the `yarn dev` workflow for CRA.) Maybe this is a good enough reason to bail on CRA, although I like the idea of being able to test the REPL without runningt he whole (slow) Jekyll build.